gf-ui-lib/src/components/SelectField/SelectField.tsx

```diff
@@ -47,7 +47,8 @@ export const SelectField = (props: SelectFieldProps) => {
 
   return (
     <div>
-      <SmallTitle
+      <Title
+        size="h4"
         text="Room type"
```

Comment:
Better don't use magic string, use props.tileText as in Numberfield

Reply:
done